cmd/lhsm-plugin-az-core/archive.go

```diff
@@ -95,25 +86,6 @@ func upload(ctx context.Context, o ArchiveOptions, blobPath string) (_ int64, er
 		return 0, err
 	}
 
-	if o.HNSEnabled && getACLResp != nil {
```

Comment:
Why are we removing this code? I thought we still wanted to preserve existing owner/group/acls?